src/server/apis/user.ts

```diff
@@ -28,8 +28,40 @@ async function handlerUserRequest(
   });
 }
 
+export async function handleRemoveRepo(
+    request: express.Request, userRecord: UserRecord): Promise<APIResponse> {
+  const owner = request.body.owner;
+  const name = request.body.name;
+
+  if (!owner || !name) {
```

Comment:
I tend to avoid these kinds of checks, while it's easier to write, the natural first question when something goes wrong is - am I missing an owner or am I missing a name.

Reply:
Fixed